rllib/core/rl_module/marl_module.py

```diff
@@ -483,7 +483,7 @@ def from_module(self, module: MultiAgentRLModule) -> "MultiAgentRLModuleSpec":
             The MultiAgentRLModuleSpec.
         """
         module_specs = {
-            module_id: SingleAgentRLModuleSpec.from_module(rl_module)
+            module_id: SingleAgentRLModuleSpec.from_module(rl_module.unwrapped())
```

Comment:
Nit: Explain why we need to unwrap here. rl_module could be a framework-specific DDP wrapper?

Reply:
done.